README.md

```diff
@@ -331,13 +333,55 @@ render(
  Returns a normalized FontFamily name where names with a space are automatically wrapped in quotes.
 
 ```js
-getFamilyName: (family: String) => string
+getFamilyName: (family: String) => String
```

Comment:
Believe this should be:

```
getFamilyName: (family: string) => string
```

ref: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Reply:
It's in the readme, done on purpose for color highlighting.

Reply:
Gotcha. Not necessarily js or TS. Is this some sort of specific documentation format, or just a convention you use?

Reply:
Just doc formatting. It's the only way I've figured out how to make them not black in color. lol